vendor/assets/stylesheets/dvl/components/sidebar_data.scss

```diff
@@ -44,15 +45,22 @@
 }
 
 .sidebar_data_icon {
-  float: left;
-  width: 1rem;
+  position: absolute;
+  top: 50%;
+  left: 0;
+  margin-top: $rhythm * -3;
+  width: $rhythm * 5;
   text-align: center;
   i {
     line-height: 1;
-    color: $darkerGray;
+    color: $gray;
   }
   @media screen and (min-width: $lapWidth) {
     width: $rhythm * 5;
+    left: $rhythm * 2;
+    .sidebar_data li:last-of-type & {
+      margin-top: $rhythm * -1.5;
```

Comment:
The last sidebar item has no bottom padding, and the first sidebar item has no top padding. So, we need to adjust the position of the icon to vertically align it properly for each item.